src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar search-bar--bg">
+      <form
+      action="#"
+      method="get"
+      class="search-bar__form"
+      data-qa="big"
+      >
```

Comment:
```suggestion
      <form
        action="#"
        method="get"
        class="search-bar__form"
        data-qa="big"
      >
```
correct indentation: 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/70469774-f937-4d7b-8b30-f4f46cbbceef)